ONSdigital/eq-author
# 275 Add fullstory service to production environment

### What is the context of this PR?

[Trello card](https://trello.com/c/r3EFP5Kc/275-add-fullstory-service)

Adds the Javascript snippet to enable FullStory service to the production environment by way of an environment variable `REACT_APP_USE_FULLSTORY`.

### How to review 

- check service is not enabled in development environments by attempting to access the `_fs_namespace` global in the browser console.
- check it is enabled in prod through the same method
